Ballot Robert

JeromeMartinez opened the pull request:
Address some issues from [Robert Wilton's ballot](https://mailarchive.ietf.org/arch/msg/cellar/v65eMDmiELzutxYyFhPXf06mUUQ/).
I don't think that I will handle other comments as I don't think we could do better, I answer to Robert about that.

JeromeMartinez commented:
PR rebased after merge of other PRs.

dericed commented on ffv1.md:
```diff
@@ -351,28 +354,20 @@ JPEG2000-RCT is a Reversible Color Transform that codes RGB (red, green, blue) P
 SVGI:![svg](rgb1.svg "rgb 1")
 SVGI:!---
-SVGC:rgb1.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\\\ g & = & Y - ( Cb + Cr ) >> 2 \\\\\\ r & = & Cr + g \\\\\\ b & = & Cb + g \\\\end{array}$$
+SVGC:rgb11.svg=$$\\\\begin{array}{ccccccc}Cb & = & b - g \\\\\\ Cr & = & r - g \\\\\\ Y & = & g + ( Cb + Cr)>>2 \\\\end{array}$$
```
The name of the svg must match in the SVGI and SVGC section, ie match the filename of the svg in lines 355 and 357 as well as 364 and 366 as well as 397 and 399 as well as 406 and 408.

dericed commented on ffv1.md:
```diff
@@ -596,6 +617,8 @@ AART:zero_state_(i) = 256 - one_state_(256-i)
 
 #### default\_state\_transition
 
+By default the following State Transition Table is used:
```
I suggest a  comma after the word `default`.

JeromeMartinez commented:
svg name issues fixed, comma added, @dericed please review again.

retokromer commented on ffv1.md:
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
+Figure: Description of the transformation of pixels from coded modified YCbCr color space to RGB color space (in case of exception). {#figureYcbcrRgbException}
+
+Background: At the time of this writing, in all known implementations of FFV1 bitstream, when `bits_per_raw_sample` was between 9 and 15 inclusive and `extra_plane` is 0, GBR Planes were used as BGR Planes during both encoding and decoding. In the meanwhile, 16-bit JPEG2000-RCT was implemented without this issue in one implementation and validated by one conformance checker. Methods to address th ...
+
```
I’m happy this will be considered.

dericed commented on ffv1.md:
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
+SVGI:![svg](rgb4.svg "rgb 4")
+SVGI:!---
+SVGC:rgb4.svg=$$\\\\begin{array}{ccccccc}b & = & Y - (Cb + Cr)>>2 \\\\\\ r & = & Cr + b \\\\\\ g & = & Cb + b \\\\end{array}$$
+AART:b = Y -(Cb + Cr) >> 2
```
add space after `-`

dericed commented on ffv1.md:
```diff
@@ -394,16 +389,40 @@ In JPEG2000-RCT, the coding order would be left to right and then top to bottom,
+SVGC:rgb3.svg=$$\\\\begin{array}{ccccccc}Cb & = & g - b \\\\\\ Cr & = & r - b \\\\\\ Y & = & b + (Cb + Cr)>>2 \\\\end{array}$$
+AART:Cb = g - b
+AART:Cr = r - b
+AART:Y = b +(Cb + Cr) >> 2
```
Add space after `+`

dericed commented on ffv1.md:
```diff
@@ -710,6 +733,8 @@ Figure: A pseudo-code description of the read of a signed integer in Golomb Rice
 |0000 0000 0001 | 11    |
 |0000 0000 0000 | ESC   |
 
+`ESC` (meaning "Escape code") case is choosen when it is considered more efficient to store the whole number compared to storing the least significant bits of the number.
```
`ESC` is already defined in Section 2.1. Is this a redefinition? If it is defined here, could it be removed from Section 2.1 so there are not competing places that define it.

dericed reviewed (commented):
Thanks. I re-reviewed and added 3 comments.

JeromeMartinez commented:
I added the missing spaces during the move of the section.
I changed the ESC commit to a move, from comments it looks like that ESC is not enough used for being worth to be at the beginning in the definitions.
@dericed please review again.

dericed commented:
thx, reviewed and tested, thanks for addressing all points.

michaelni commented:
d1d3da1efb89efc96f5207cbf4ffcd37cb10aa06 adds a line thats removed in the next commit

JeromeMartinez commented:
> d1d3da1 adds a line thats removed in the next commit

Fixed, please review again / merge.
